tests/dist/chats.spec.dev.js

```diff
@@ -293,37 +302,38 @@ describe('chat resolvers', function () {
   test('send message to group chat', function _callee6() {
     var sendMessageToGroupChatResponse, sendMessageToAGroupChatData;
     return regeneratorRuntime.async(function _callee6$(_context6) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context6.prev = _context6.next)) {
           case 0:
             _context6.next = 2;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n        mutation{\n          sendMessageToGroupChat(chatId: "'.concat(
-                  createdGroupChatId,
-                  '", messageContent: "this is a test message"){\n            _id\n          }\n        }\n            '
-                ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n        mutation{\n          sendMessageToGroupChat(chatId: "'.concat(
+                    createdGroupChatId,
+                    '", messageContent: "this is a test message"){\n            _id\n          }\n        }\n            '
+                  ),
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.